resources/configs/simple_flop_config.json

```diff
@@ -0,0 +1,55 @@
+{
+  "card_config": {
+    "flop": "6h9dTd",
```

Comment:
When we save the config from batch solves, we should probably not include `"flop"` in the config, since there could be multiple flops, and it's also captured by the separate `boards.txt` output (not sure if that was the name we used).

Alternatively, we could output a _list_ of flops here within the card config, but that would beg us to redesign the actual `CardConfig` struct to contain a list of flops, which is not something that sounds appealing.

Reply:
See discussion at top of PR: sometimes we want to include the flop. We don't know at save time whether or not we want the flop. Basically we just include dummy cards, and if the user wants to overwrite this then they can (via --boards or --board-files).

I'd be open to making flop/turn/river optional (turn and river currently are actually, could make flop optional as well). But they should definitely be able to specify flops if they'd like.